tracing/tracing.go

```diff
@@ -0,0 +1,55 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/HomesNZ/env"
+	"github.com/pkg/errors"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/exporters/trace/jaeger"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+// TracerConfig manages initialization of the tracing collector
+type TracerConfig struct {
+	Name          string
+	CollectorHost string
+	CollectorPort string
+	CollectorPath string
+}
+
+func (cfg TracerConfig) collectorEndpoint() string {
+	return fmt.Sprintf("http://%s:%s%s", cfg.CollectorHost, cfg.CollectorPort, cfg.CollectorPath)
```

Comment:
I would really prefer if you use https://golang.org/pkg/net/url/ here instead of random sprintf formatting. 
It has all methods there : Path, Host, Port and Scheme(also, why it's http?)